application/.gitignore

```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+# JetBrains local files
+/.idea/
+
```

Comment:
```suggestion

```
Remove this again. The .idea folder contains some relevant data to the project. As you are using IntelliJ please refer to existig code and compare which parts of the .idea folder are contained in the project.

Additionally the .idea folder has its own .gitignore that i.e. defines that the workspace.xml will be ignored.

Look over what IntelliJ wants to commit in your .idea folder (there shouldn't be anything changing in this folder as it mostly contains static config files)